package-lock.json

```diff
@@ -12983,8 +12983,7 @@
     },
     "node_modules/terraso-backend": {
       "version": "0.1.0",
-      "resolved": "git+ssh://git@github.com/techmatters/terraso-backend.git#8813449c8b2f4eb4b70b4add87007d2a93d25baa",
-      "integrity": "sha512-WXumMW1dnbEFfZjmNkRuibIuO7TDeRUQ8vr5K2nyhUs8bDZXUS67zM6TtLx92czO+PvdzI36ncH5abGHg3rpOQ=="
+      "resolved": "git+ssh://git@github.com/techmatters/terraso-backend.git#d87539e82c9ae59ace3948e1350f14ec25fedce4"
```

Comment:
question: Any idea why this changed?

Reply:
i guess because it did not get updated properly in this most recent change on `main`: https://github.com/techmatters/terraso-client-shared/commit/a0e3a5eac8a0f051c78ae54544502d0ebd7dab4d